### PR TITLE
Added NoEmptyCommentFixer

### PR DESCRIPTION
**Pull request by SpacePossum**
Fixer to remove empty (i.e. white space only) comments.

### CODE REVIEW

File: Symfony/CS/Fixer/Contrib/NoEmptyCommentFixer.php [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+        if ('#' === $content[0]) {
+            if (preg_match('/^#\s*$/', $content)) {
+                $this->clearCommentToken($tokens, $index);
+            }
+
+            return;
+        }
+
+        if ('/' === $content[1]) {
+            if (preg_match('#^//\s*$#', $content)) {
```
Comment (keradus) [inline Symfony/CS/Fixer/Contrib/NoEmptyCommentFixer.php]: I would also clear `////` and `##`

File: Symfony/CS/Tests/Fixtures/Integration/priority/phpdoc_to_comment,no_empty_comment.test [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+}
+--INPUT--
+<?php
+
+$first = true;// needed because by default first docblock is never fixed.
+
+/** */
+foreach($connections as $key => $sqlite) {
+    $sqlite->open($path);
+}
```
Comment (keradus) [inline Symfony/CS/Tests/Fixtures/Integration/priority/phpdoc_to_comment,no_empty_comment.test]: please add missing line

File: Symfony/CS/Fixer/Contrib/NoEmptyCommentFixer.php [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+                $this->clearCommentToken($tokens, $index);
+            }
+
+            return;
+        }
+
+        // comment of type /*[]*/
```
Comment (keradus) [inline Symfony/CS/Fixer/Contrib/NoEmptyCommentFixer.php]: what type? I can't get it

File: Symfony/CS/Fixer/Contrib/NoEmptyCommentFixer.php [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+        }
+    }
+
+    /**
+     * Clear comment token, but preserve trailing linebreak if there is any.
+     *
+     * @param Tokens $tokens
+     * @param int    $index  T_COMMENT index
+     *
+     * @deprecated Will be removed in the 2.0.
```
Comment (keradus) [inline Symfony/CS/Fixer/Contrib/NoEmptyCommentFixer.php]: thank you for that notice !

File: Symfony/CS/Fixer/Contrib/NoEmptyCommentFixer.php [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+
+    /**
+     * @param Tokens $tokens
+     * @param int    $index  T_COMMENT index
+     */
+    private function fixComment(Tokens $tokens, $index)
+    {
+        $content = $tokens[$index]->getContent();
+        if ('#' === $content[0]) {
+            if (preg_match('/^#\s*$/', $content)) {
```
Comment (keradus) [inline Symfony/CS/Fixer/Contrib/NoEmptyCommentFixer.php]: Please, use same delimiters in one class if possible (here, it is ;) ) - eg `{...}`

**Comment by keradus** [comment]
few minors and good to go ;)

**Comment by keradus** [comment]
ping @SpacePossum

File: Symfony/CS/Fixer/Contrib/NoEmptyCommentFixer.php (hunk 4 above)
Comment (GrahamCampbell) [inline Symfony/CS/Fixer/Contrib/NoEmptyCommentFixer.php]: Why deprecate it? Just delete it. It's a private function?

Comment (keradus) [inline Symfony/CS/Fixer/Contrib/NoEmptyCommentFixer.php]: That's the point, it's needed on 1.12 and need to be removed on 2.x

Comment (GrahamCampbell) [inline Symfony/CS/Fixer/Contrib/NoEmptyCommentFixer.php]: Oh, I see. Thanks for the clarification. ;)

**Comment by keradus** [comment]
Thank you @SpacePossum.

**Pull request closed without merging**